os/dbuild.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Checking docker is installed
+which docker > /dev/null 2>&1
+HAS_DOCKER=$?
+docker --version > /dev/null 2>&1
+HAS_VALID_DOCKER=$?
+
+SYSTEM_OK=false
+if [ $HAS_DOCKER -eq 0 ]; then
+    if [ $HAS_VALID_DOCKER -eq 0 ]; then
+        SYSTEM_OK=true
+    fi 
+fi
+
+if [ $SYSTEM_OK != "true" ]; then
+    echo " DEPENDENCY ERROR"
+	echo
+	echo "You don't have docker installed in your path."
+	echo "Please install docker client."
+	echo "You can find it here:"
+	echo
+	echo "   https://docs.docker.com/install/linux/docker-ce/ubuntu/"
+	echo
+    exit 1
+fi
+
+# Run TizenRT docker container
+BUILD_CMD=make
+BUILD_OPT=$1
+BUILD_OPT_LOWER=$(echo "$BUILD_OPT" | tr '[:upper:]' '[:lower:]')
+
+case "$BUILD_OPT_LOWER" in
+download )
+	${BUILD_CMD} ${BUILD_OPT_LOWER} $2
+	;;
+*)
+	docker rm -f tizenrt > /dev/null 2>&1
+	docker run --rm -it -d -v $(pwd)/..:/root/tizenrt --name tizenrt -w /root/tizenrt/os tizenrt/tizenrt /bin/bash
+	docker exec -it tizenrt ${BUILD_CMD} ${BUILD_OPT_LOWER}
+	docker stop tizenrt > /dev/null 2>&1
+	;;
+esac
+
```

Comment:
Here is no usages tab.
Even though we guide commands on our quick start, we need to support ```--help``` command to show usages.

Reply:
@sunghan-chang 
I think the usage of dbuild.sh would be quite long because it includes several commands such as clean, distclean, menuconfig, build, download.
Let's add the usage later after preparing it with careful consideration.

Reply:
You already prepare them in README.
download, menuconfig, clean, distclean are enough.
Because developer does not see README carefully, supporting ```--help``` is important.